src/promisified-get-request.js

```diff
@@ -0,0 +1,32 @@
+import https from 'https';
```

Comment:
I propose to rename this file as `https-request` and rename function to `httpsRequest`.